pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -146,7 +146,7 @@
             doc = "Number of threads to use for HTTP requests processing"
                 + " Default is set to `2 * Runtime.getRuntime().availableProcessors()`"
         )
-    private int numHttpServerThreads = 2 * Runtime.getRuntime().availableProcessors();
+    private int numHttpServerThreads = Math.max(4, 2 * Runtime.getRuntime().availableProcessors());
```

Comment:
Is 4 enough? I've seen cases were I would still complain at 4.

Reply:
What do you think would be the right number?
One thing is that with the current settings, on a one core machine, pulsar standalone wouldnt even come up. This fix solves that.

Reply:
I've been putting it at 16 to get around that issue, though admittedly I pulled that number out of thin air.